bundles/org.openhab.binding.automower/README.md

```diff
@@ -34,23 +34,31 @@ With the default value of 1h this would mean ~720 requests per month for the bri
+|-------------------------|----------|-------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| mower-status#mode                     | String   | R | The current mode (MAIN_AREA, SECONDARY_AREA, HOME, DEMO, UNKNOWN)                                                                                                |
+| mower-status#activity                 | String   | R | The current activity (UNKNOWN, NOT_APPLICABLE, MOWING, GOING_HOME, CHARGING, LEAVING, PARKED_IN_CS, STOPPED_IN_GARDEN)                                           |
+| mower-status#state                   | String   | R | The current state (UNKNOWN, NOT_APPLICABLE, PAUSED, IN_OPERATION, WAIT_UPDATING, WAIT_POWER_UP, RESTRICTED_NONE, RESTRICTED_WEEK_SCHEDULE, RESTRICTED_PARK_OVERRIDE, RESTRICTED_SENSOR, RESTRICTED_DAILY_LIMIT, OFF, STOPPED, ERROR, FATAL_ERROR, ERROR_AT_POWER_UP)  |
+| mower-status#last-update              | DateTime | R | The time when the automower updated its states                                                                                                                   |
+| mower-status#battery                  | Number   | R | The battery state of charge in percent                                                                                                                           |
+| mower-status#error-code               | Number   | R | The current error code                                                                                                                                           |
+| mower-status#error-timestamp         | DateTime | R | The timestamp when the current error occurred                                                                                                                    |
+| mower-status#planner-next-start      | DateTime | R | The time for the next auto start. If the mower is charging then the value is the estimated time when it will be leaving the charging station. If the mower is about to start now, the value is NULL.                                                                  |
+| mower-status#planner-override-action | String   | R | The action that overrides current planner operation.                                                                                                                                                                                                                  |
+| mower-status#calendar-tasks          | String   | R | The JSON with the information about Automower planner.                                                                                                    |
+| mower#start                       | Number   | W | Starts the automower for a duration                     |
```

Comment:
Cool, I like the new command channels. I tried to also do command channels in my initial contribution but they were rejected. But probably because of the way that I implemented them, they didn't fit well and were more like a workaround because I wanted to use actions.